From 1893dba8a172636791dc79137a3331bd41cc50bc Mon Sep 17 00:00:00 2001
From: Patric Stout <truebrain@openttd.org>
Date: Sat, 2 Apr 2022 12:36:29 +0200
Subject: [PATCH 0822/1053] Fix: [Actions] Remove Ubuntu:groovy from release
 pipeline (#9846)

It no longer exists upstream. Let's focus on LTS only for now.
---
 .github/workflows/release.yml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/.github/workflows/release.yml b/.github/workflows/release.yml
index fd6a8d8fa1..d3b4dc329b 100644
--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -377,9 +377,6 @@ jobs:
         - container_image: "ubuntu:20.04"
           bundle_name: "focal"
           compiler: "g++"
-        - container_image: "ubuntu:20.10"
-          bundle_name: "groovy"
-          compiler: "g++"
         - container_image: "debian:buster"
           bundle_name: "buster"
           compiler: "g++"
-- 
2.55.0

